contracts/test/_fixture.js

```diff
@@ -494,42 +532,148 @@ async function convexVaultFixture() {
 async function convexMetaVaultFixture() {
   const fixture = await loadFixture(defaultFixture);
 
-  const { governorAddr } = await getNamedAccounts();
-  const sGovernor = await ethers.provider.getSigner(governorAddr);
+  if (isFork) {
+    const { josh, matt, anna, domen, daniel, franck, ousd } = fixture;
 
-  // Add Convex Meta strategy
-  await fixture.vault
-    .connect(sGovernor)
-    .approveStrategy(fixture.OUSDmetaStrategy.address);
+    // const curveFactoryAddress = '0xB9fC157394Af804a3578134A6585C0dc9cc990d4'
 
-  // set meta strategy on vault so meta strategy is allowed to mint OUSD
-  await fixture.vault
-    .connect(sGovernor)
-    .setOusdMetaStrategy(fixture.OUSDmetaStrategy.address);
+    const threepoolLP = await ethers.getContractAt(
+      threepoolLPAbi,
+      addresses.mainnet.ThreePoolToken
+    );
+    const ousdMetaPool = await ethers.getContractAt(
+      ousdMetapoolAbi,
+      addresses.mainnet.CurveOUSDMetaPool
+    );
+    const threepoolSwap = await ethers.getContractAt(
+      threepoolSwapAbi,
+      addresses.mainnet.ThreePool
+    );
+    // const curveFactory = await ethers.getContractAt(curveFactoryAbi, curveFactoryAddress)
+
+    // Get some 3CRV from most loaded contracts/wallets
+    await transferTokensFromContract(
+      addresses.mainnet.ThreePoolToken,
+      [
+        "0xceaf7747579696a2f0bb206a14210e3c9e6fb269",
```

Comment:
These addresses are Curve metapools (Frax, USD, alUsd, Lusd & another one). Would be cool not transfering all the 3crv tokens from that contract -- and I am only saying this because I fell into that pitfall before :) Maybe transfering 30% of the tokens out would be enought.

The issues is when you start fork testing those contract and they have no 3Crv (and we might fork test Frax & Lusd soon for example) then unexpected errors happen because those contracts expect to have 3Crv balance but in reality have none.

Reply:
Yep, sounds good to me, making that change